app/src/app.tsx

```diff
@@ -149,24 +149,31 @@ export const App = (props: IAppProps): React.ReactElement => {
     }
   }, [chainId, web3]);
 
+  const routes: IRoute[] = [
+    { default: true,
+      path: '/',
+      page: HomePage,
+      subRoutes: [{ path: '/tokens/:tokenId',
+        page: TokenPage,
+        subRoutes: [
```

Comment:
these should not be subroutes - it should be based on how it was in the old code - only homepage had subroutes